sampleconfig/core.yaml

```diff
@@ -53,6 +53,9 @@ peer:
         # endorsementTimeout is the duration the gateway waits for a response
         # from other endorsing peers before returning a timeout error to the client.
         endorsementTimeout: 30s
+        # broadcastTimeout is the duration the gateway waits for a response
+        # from ordering nodes before returning a timeout error to the client.
```

Comment:
Is this description correct? Or is it a timeout before the Gateway gives up on a specific orderer and tries another one?